<launch>

  <!-- cascaded pid control node -->
  <node pkg="cascaded_pid_control" type="cascaded_pid_control" name="CascadedPidControl" output="screen">
    <remap from="~odometry" to="/CheatNode/odometry" />
    <remap from="~rateThrust" to="/uav/input/rateThrust" />
  </node>

  <!-- cheat node -->
  <node pkg="cascaded_pid_control" type="cheat_node" name="CheatNode" output="screen" />

</launch>